Overview.src.html

```diff
@@ -4027,6 +4039,51 @@ <h3 id=http-cors-protocol>CORS protocol</h3>
  <li><p>Return <var title>p</var>.
 </ol>
 
+<h3>Fetch termination initiated by the user agent</h3>
+
+<p>The user agent may <span title=concept-fetch-terminate>terminate</span> an ongoing fetch with
+reason <i title>garbage collection</i> if that termination is not observable through script.
+
+<p class="note no-backref">The server observing garbage collection has precedent, e.g. with
+WebSocket and XMLHttpRequest.
+
+<div class="example no-backref">
+ <p>The user agent can terminate the fetch because the termination cannot be observed.
+ <pre>fetch("https://www.example.com/")</pre>
+</div>
+
+<div class="example no-backref">
+ <p>The user agent cannot terminate the fetch because the termination can be observed through
+ the promise.
+ <pre>window.promise = fetch("https://www.example.com/")</pre>
+</div>
+
+<div class="example no-backref">
+ <p>The user agent can terminate the fetch because the associated body is not observable.
+ <pre>window.promise = fetch("https://www.example.com/").then(res => res.headers)</pre>
+</div>
+
+<div class="example no-backref">
+ <p>The user agent can terminate the fetch because the termination cannot be observed.
+ <pre>fetch("https://www.example.com/").then(res => res.body.getReader().closed)</pre>
+</div>
+
+<div class="example no-backref">
+ <p>The user agent cannot terminate the fetch because one can observe the termination by registering
+ a handler for the promise object.
+ <pre>window.promise = fetch("https://www.example.com/")
+  .then(res => res.body.getReader().closed)</pre>
+</div>
+
+<div class="example no-backref">
+ <p>The user agent cannot terminate the fetch as termination would be observable via the registered
+ handler.
+ <pre>fetch("https://www.example.com/")
+  .then(res => {
+    res.body.getReader().closed.then(() => console.log("stream closed!"))
+  })</pre>
+</div>
```

Comment:
All these examples can use a single `<div class="example no-backref">`. No need to have one per example.